src/bitcoin-cli.cpp

```diff
@@ -25,6 +25,7 @@
 #include <functional>
 #include <memory>
 #include <optional>
+#include <stdexcept>
```

Comment:
Why this additional include? It isn't related to the removal of libevent compatibility code. If because of `std::runtime_error` use below, should be a separate commit.

Reply:
Removed.